fbuapp/View Controllers/ProfileViewController.m

```diff
@@ -23,18 +23,31 @@ @implementation ProfileViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    
+    //Set current user's information to their profile page.
+    PFUser *currentUser = [PFUser currentUser];
+    if (currentUser != nil) {
```

Comment:
maybe add an "else" with NSLog to show that there's an error/shouldn't have reached profileviewcontroller if PFuser is nil